### PR TITLE
qwen agent local support

### PR DESCRIPTION
**Pull request by Zhikaiiii**
1. support qwen-agent local use one local llm
2. move text-to-image to sd-xl

### CODE REVIEW

File: modelscope_agent/llm/llm_factory.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from .local_llm import LocalLLM
+from .ms_gpt import ModelScopeGPT
+from .openai import OpenAi
+from .qwen import QWen
+
+LLM_MAPPING = {'ms_gpt': ModelScopeGPT, 'openai': OpenAi}
```
Comment (zzhangpurdue) [inline modelscope_agent/llm/llm_factory.py]: add dependency call into dict as well。 such as
  if type == 'ms_gpt':
        from .ms_gpt import ModelScopeGPT
        return ModelScopeGPT
    elif ...

File: modelscope_agent/llm/llm_factory.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class LLMFactory:
+
+    @staticmethod
+    def build_llm(name, cfg):
+        llm_cls = LLM_MAPPING.get(name, LocalLLM)
+        llm_cfg = cfg.get(name, {})
```
Comment (zzhangpurdue) [inline modelscope_agent/llm/llm_factory.py]: 'name' changes to 'type'

File: modelscope_agent/llm/llm_factory.py (hunk 1 above)
Comment (zzhangpurdue) [inline modelscope_agent/llm/llm_factory.py]: so, what about other opensource LLM?

Comment (Zhikaiiii) [reply modelscope_agent/llm/llm_factory.py]: 这样的话，后续有新的LLM，build_llm函数可能需要一直改动？

Comment (zzhangpurdue) [reply modelscope_agent/llm/llm_factory.py]: 我的意思是这样子：
`def get_llm_cls():
   if type == 'ms_gpt':
      from .ms_gpt import ModelScopeGPT
      return ModelScopeGPT
   elif type == 'openai':
      from .openai import OpenAi
      return OpenAi

   llm_cls = get_llm_cls(type)
`

File: README.md [hunk 3]
```diff
@@ -17,9 +20,39 @@ llm = ModelScopeGPT(model_cfg)
...
 
-agent.run('写一个 2023 上海世界人工智能大会 20 字以内的口号，并念出来')
 ```
 
+- Single-step tool-use
+
+```Python
+agent.run('使用地址识别模型，从下面的地址中找到省市区等元素，地址：浙江杭州市江干区九堡镇三村村一区', remote=True)
+```
+![image](resource/modelscopegpt_case_single-step.png)
```
Comment (wenmengzhou) [inline README.md]: is this the result image of the opensource qwen agent llm

**Review by zzclynn — APPROVED** [review]

**Pull request merged**